src/man/sssd.conf.5.xml

```diff
@@ -4571,6 +4571,17 @@ ldap_user_extra_attrs = phone:telephoneNumber
                     to log in either only with the password or with both factors
                     two-step prompting has to be used.
                     </para>
+                    <para>
+                    Please note the some clients, e.g. ssh with
+                    'PasswordAuthentication yes' will generate the prompts on
+                    their own and will not use prompts provided by SSSD or
+                    other PAM modules. Additionally with respect to ssh and
+                    PasswordAuthentication please note that if two-factor
+                    authentication is available SSSD will assume that the
+                    credentials given by the user at the ssh password prompt
+                    will always be the two factors in a single string even if
+                    two-factor authentication is optional.
```

Comment:
```suggestion
                    <para>
                    Some clients, such as SSH with
                    'PasswordAuthentication yes', generate their own prompts
                    and do not use prompts provided by SSSD or other PAM
                    modules. Additionally, for SSH with
                    PasswordAuthentication, if two-factor authentication is
                    available, SSSD expects that the
                    credentials entered by the user at the SSH password prompt
                    will always be the two factors in a single string, even if
                    two-factor authentication is optional.
```

@sumit-bose I hope some parts of the text are useful but I'm pretty sure the indentation in my suggestion is wrong :)

Reply:
Hi,

thank you, I have included your suggestion in the latest version.

bye,
Sumit